core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -118,12 +119,14 @@ private object GroupMetadata {
                 protocolType: String,
                 protocol: String,
                 leaderId: String,
+                currentStateTimestamp: Option[Long],
                 members: Iterable[MemberMetadata]): GroupMetadata = {
-    val group = new GroupMetadata(groupId, initialState)
+    val group = new GroupMetadata(groupId, initialState, Time.SYSTEM)
```

Comment:
We should avoid hard-coding references to `Time.SYSTEM` when possible. Maybe we can pass the time instance as another argument?